src/pages/Beta/Swap/WatchList/CurrencyPopover/index.tsx

```diff
@@ -70,6 +70,15 @@ const CurrencyPopover: React.FC<Props> = ({
     if (searchToken) return [searchToken]
     const sorted = filteredTokens.sort(tokenComparator)
 
+    // if avax in sorted tokens, move it to the top
+    const containAvax = sorted.some(token => token.symbol === 'WAVAX')
+    if (containAvax) {
+      const avaxIndex = sorted.findIndex(token => token.symbol === 'WAVAX')
```

Comment:
shouldn't this re-sort be addressed as a parm in `useTokenComparator`?

if we are indeed modifying the data here, we can avoid iterating `sorted` twice by something like:
```javascript
const avaxIndex = sorted.findIndex(token => token.symbol === 'WAVAX')
if (avaxIndex > 0) { // Index of 0 means it is already first
    const avax = filteredTokens[avaxIndex]
    sorted.splice(avaxIndex, 1)
    sorted.unshift(avax)
}
```